integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -76,11 +76,15 @@ public void testJaxbAnnotationTransfer() {
     }
 
     /**
-     * This test is disabled in native mode as there is no interaction with the quarkus integration test endpoint.
+     * _PanacheEntityBase_ has the method _isPersistent_. This method is used by jackson to serialize the attribute *persistent*
```

Comment:
I am not sure if this is too much detail or not. I'll let @FroMage decide

Reply:
Maybe we could go just with this explanation.
```java
@DisabledOnNativeImage("Disabled in the native mode as there is no interaction with any endpoint")
```

Reply:
@rsvoboda This is valid. I described the reason for the test as there is no issue ticket linkable.
I do not know how much detail to put in there...

Reply:
@FroMage Is this to much information in the comment?

Reply:
It's long and not super useful, but it doesn't hurt so whatever :)

Reply:
Thx!